packages/editor/src/client/locales/en.json

```diff
@@ -125,7 +125,13 @@
         "postImage": "Image for Post",
         "metadataInfo": "Information from these fields are being used for previews in Social Media Posts. If left blank, information from general-tab will be used instead.",
         "lightStyledWithNoImageError": "Light styled teasers must have an image",
+        "titleHelpBlock": "Please use an SEO-optimised title",
+        "leadHelpBlock": "Please use an SEO-optimised lead",
+        "charCount": "Characters count {{charCount}}",
+        "charCountWarning": "The maximum number of characters should be",
+        "totalCharCount": "Total characters count {{charCount}}",
         "seoTitleHelpBlock": "Please use an SEO-optimised title"
```

Comment:
Can this be removed?

Reply:
Why ? It is used in the metaDataPanel of the article.